index.bs

```diff
@@ -113,186 +113,207 @@ div.enum-description > table > thead > tr > th {
+      a media element, then querying by that media element would become
+      {{allowed}} if a user-agent decides to bless that element because that
+      behavior seems intended by users, but querying by a contructor and querying
+      by other media elements, which haven't been clicked yet, would still
+      return {{disallowed}}.
     </div>
 
     <div class="note">
-      Note: If the document has child documents, then the result from child
-      documents could be different from the result of their parent documents
-      depending on the implementation.
+      Note: Depending on the implementation, if a document has child
+      documents, then the result queried from the {{Navigator}} object
+      asscociated with the parent document could be different from the result
+      queried from the {{Navigator}} object asscociated with the child
+      documents.
     </div>
 
     <div class=example>
       Assume that the top level document A in `foo.com` returns {{allowed}} and
       it has an embedded iframe, which has another document B from `bar.com`. A
-      user-agent could either make child document B return the same result that
-      is inherited from the top level document A. Or make the document B return
-      a different result, eg. {{disallowed}}.
+      user-agent could either make child document B return same result that is
+      inherited from the top level document A. Or make the document B return a
+      different result, eg. {{disallowed}}.
 
       Doing the former helps to lower the complexity and make the behavior of
       blocking autoplay more consistent. The latter helps providing a
       finer-grained autoplay control.
     </div>
 
-    <div class="note">
-      Note: In addition, document's {{Document/autoplayPolicy}} can also be
-      used to know whether [[webaudio inline]] can be allowed to start if a
-      user-agent blocks web audio by default.
-    </div>
-
-    <div class=example>
-      [Web Audio](https://webaudio.github.io/web-audio-api/#allowed-to-start)
-      uses [=sticky activation=] to determine if {{AudioContext}} can be allowed
-      to start. If the document is allowed to autoplay, then {{AudioContext}}
-      should also be allowed to start.
-
-      <pre class="lang-javascript">
-        var ac = new AudioContext;
-        if (document.autoplayPolicy === "allowed") {
-          ac.onstatechange = function() {
-            if (ac.state === "running") {
-              // Start running audio app
-            } else {
-              // Display a bit of UI to ask the user to start the audio app.
-              // Audio starts via calling ac.resume() from a handler, and
-              // 'onstatechange' allows knowing when the audio stack is ready.
-            }
-          }
-        }
-      </pre>
-    </div>
-
-  <h3 id="media-element-api">The HTMLMediaElement API</h3>
+  <h4 id="query-by-a-element">Query by a Element</h3>
```

Comment:
I only spotted a few typos like this. I think it should be 'an element' instead of 'a element'.

```suggestion
  <h4 id="query-by-an-element">Query by an Element</h3>
```

A similar typo is also hiding here ...

https://github.com/w3c/autoplay/pull/16/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985R264

... and here ...

https://github.com/w3c/autoplay/pull/16/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985R312